pages/api/volunteerProgramApplications/create.ts

```diff
@@ -0,0 +1,29 @@
+import dbConnect from 'lib/dbConnect';
+import { hash } from 'bcrypt';
+import type { NextApiRequest, NextApiResponse } from 'next';
+import Users from 'models/Users';
+import VolunteerProgramApplications from 'models/VolunteerProgramApplications';
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<any>
+) {
+    
+  switch (req.method) {
+    case 'POST':
+        const {userId, isApproved, emergencyContact, workStatus, employer, opportunites} = req.body;
+        const status = await VolunteerProgramApplications.create({
```

Comment:
you need to add `await dbConnect()` to connect to the database before this line :)